p2p/cuprate-p2p/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+//! Cuprate's P2P Crate.
+//!
+//! This crate contains a [`ClientPool`](client_pool::ClientPool) which holds connected peers on a single [`NetworkZone`](monero_p2p::NetworkZone).
+//!
+//! This crate also contains the different routing methods that control how messages should be sent, i.e. broadcast to all,
+//! or send to a single peer.
+//!
+#![allow(dead_code)]
+
```

Comment:
Is this still needed?

Reply:
yeah, not everything is public, it will be removed though in a future PR